agents/rust/mediator/tests/common/agent_and_transport_utils.rs

```diff
@@ -68,15 +74,57 @@ pub async fn send_message_and_pop_response_message(
     our_verkey: &VerKey,
     their_diddoc: &AriesDidDoc,
 ) -> Result<String> {
-    agent
-        .pack_and_send_didcomm(message_bytes, our_verkey, their_diddoc, aries_transport)
+    // Wrap message in encrypted envelope
+    let EncryptionEnvelope(packed_message) = agent
+        .pack_didcomm(message_bytes, our_verkey, their_diddoc)
         .await
-        .map_err(|err| GenericStringError { msg: err })?;
+        .map_err(|e| GenericStringError { msg: e.to_string() })?;
+    let packed_json = serde_json::from_slice(&packed_message)?;
+    info!(
```

Comment:
Log as part of `send_aries_envelope`, but I'd suggest at least debug (or lower)

Reply:
Done.